redash/worker.py

```diff
@@ -79,3 +79,12 @@ def __call__(self, *args, **kwargs):
 def init_celery_flask_app(**kwargs):
     app = create_app()
     app.app_context().push()
+
+
+# Hook for extensions to add periodic tasks.
+@celery.on_after_configure.connect
+def add_periodic_tasks(sender, **kwargs):
+    app = create_app()
+    periodic_tasks = getattr(app, 'periodic_tasks', {})
```

Comment:
Where `periodic_tasks` is being set?

Reply:
`periodic_tasks` would get set inside the `create_app()` call by an extension.

For example, here is an extension that does this: https://github.com/mozilla/redash-stmo/blob/master/src/redash_stmo/health.py#L105

Reply:
I see. I think we need a more defined/strict contract between the extensions and the app. Otherwise it's too easy to shoot yourself in the foot -- for example here, some extension might override `periodic_tasks` when initializing instead of assigning an empty value only when empty.

Not a concern right now, but we should address this before releasing the extensions API to the "public".